src/reducers/real/index.js

```diff
@@ -0,0 +1,90 @@
+import {
+  ITEM_ADD,
+  ITEM_DELETE,
+  ITEM_RENAME,
+  CONTAINER_ADD,
+  CONTAINER_DELETE,
+  CONTAINER_RENAME,
+  SET_REAL_DATA
+} from 'actions/actiontypes'
+
+const initialState = {
+  _id: '',
+  name: '',
+  items: [],
+  modified_timestamp: '',
+  containers: [],
+  is_deleted: false,
+  timestamp: '',
+  snapshots: [],
+  unsnapshot_items: []
+}
+
+const realReducer = (state = initialState, action) => {
+  switch (action.type) {
+    case ITEM_ADD:
```

Comment:
can we write a test that tests that when an item is added to an arrangement all the snapshots have it added to the "_unassigned" container. The "_unassigned" container should have a size of "-1" which means infinite.